macos/FluentUI/Button/Button.swift

```diff
@@ -188,6 +188,7 @@ open class Button: NSButton {
 	}
 
 	open override func mouseUp(with event: NSEvent) {
+		_ = target?.perform(action, with: event)
```

Comment:
Should we be doing this by calling super in these overrides instead so we inherit superclass behavior instead of recreating it?

Reply:
I think so. As I'm doing more testing, calling perform action manually in mouseUp() introduces bugs like button action fires on click-drag-release gesture.

Reply:
For some reason when I call super.mouseDown, contentTintColor doesn't render as expected.
![fluent_button](https://user-images.githubusercontent.com/67026167/103254555-bef87800-4953-11eb-95da-84e75b65abb9.gif)
(investigating)

Reply:
Reversing the order might help:

`super.mouseDown(with: event)`
`mouseDown = true`

it's the assignment `mouseDown = true` that sets the contentTintColor, so it would override anything that the `super` call did.

Reply:
super.mouseDown blocks until the button is released. So if we call updateContentTintColor() after mouseDown, contentTintColor will not update while button is in pressed state.